<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>configureDialog</class>
 <widget class="QDialog" name="configureDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>702</width>
    <height>492</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>configuration window</string>
  </property>
  <widget class="QGroupBox" name="gbUpdateInterval">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>661</width>
     <height>131</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>461</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>Set the time interval btw updates.Default and recommended interval is 1 day, 0 hours, 0 minutes</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QComboBox" name="gbUpdateIntervalComboBoxDays">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>30</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>3</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>4</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>5</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>6</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>7</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="gbUpdateIntervalComboBoxHours">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>60</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>3</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>4</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>5</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>6</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>7</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>8</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>9</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>10</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>11</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>12</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>13</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>14</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>15</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>16</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>17</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>18</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>19</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>20</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>21</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>22</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>23</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>24</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="gbUpdateIntervalComboBoxMinutes">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>90</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>3</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>4</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>5</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>6</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>7</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>8</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>9</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>10</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>11</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>12</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>13</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>14</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>15</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>16</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>17</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>18</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>19</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>20</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>21</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>22</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>23</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>24</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>25</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>26</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>27</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>28</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>29</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>30</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>31</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>32</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>33</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>34</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>35</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>36</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>37</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>38</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>39</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>40</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>41</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>42</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>43</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>44</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>45</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>46</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>47</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>48</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>49</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>50</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>51</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>52</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>53</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>54</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>55</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>56</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>57</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>58</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>59</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>60</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="labelDays">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>30</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>day(s)</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelHours">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>60</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>hour(s)</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelMinutes">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>90</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>minute(s)</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="gbDelayStartInterval">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>661</width>
     <height>101</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>241</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>461</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>When the program starts, wait for this number of minutes before checking for updates (useful to give time to set up network interfaces)</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QComboBox" name="gbDelayStartIntervalComboBox">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>40</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>5</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>10</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>15</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>30</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QGroupBox" name="gbAutoStart_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>410</y>
     <width>661</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>461</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Select localization language</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxLanguageList">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>20</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pbClose">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>460</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;Close</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="gbAutoStart">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>340</y>
     <width>661</width>
     <height>71</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>-20</y>
      <width>241</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>461</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Auto start application at login</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxAutoStart">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>20</y>
      <width>181</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="gbSynapticAutoRefresh">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>240</y>
     <width>661</width>
     <height>101</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="labelSynapticAutoRefresh">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>461</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>Auto reload Synaptic when it is started from the context menu (Synaptic may crash under certain use cases)</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxSynapticAutoRefresh">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>20</y>
      <width>171</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>gbUpdateIntervalComboBoxDays</tabstop>
  <tabstop>gbUpdateIntervalComboBoxHours</tabstop>
  <tabstop>gbUpdateIntervalComboBoxMinutes</tabstop>
  <tabstop>gbDelayStartIntervalComboBox</tabstop>
  <tabstop>checkBoxSynapticAutoRefresh</tabstop>
  <tabstop>checkBoxAutoStart</tabstop>
  <tabstop>comboBoxLanguageList</tabstop>
  <tabstop>pbClose</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
